provisioning/README.md

```diff
@@ -0,0 +1,370 @@
+| ONTOP_CORS_ALLOWED_ORIGINS  |           | * (default)                                                            | CORS domain name                     |      |
+| ONTOP_DEV_MODE              |           | true (default)                                                         | Redeploy endpoint on file changes    | X    |
+
+Here is an example which exposes two endpoints for two different roles (database users, restricted mappings but same ontology)
+
+```console
+docker run -p 8080:8080 -p 8082:8082 \
+  -v $(pwd)/resources/university.ttl:/input/ontology.ttl \
+  -v $(pwd)/resources/university-role1.obda:/input/role1.obda \
+  -v $(pwd)/resources/university-role1.properties:/input/role1.properties \
+  -v $(pwd)/resources/university-role2.obda:/input/role2.obda \
+  -v $(pwd)/resources/university-role2.properties:/input/role2.properties \
+  -v $(pwd)/resources/university.sql:/tmp/university.sql \
+  -e ONTOP_PORT="8080 8082" \
+  -e ONTOP_ONTOLOGY_FILE="/input/ontology.ttl /input/ontology.ttl" \
+  -e ONTOP_MAPPING_FILE="/input/role1.obda /input/role2.obda" \
+  -e ONTOP_PROPERTIES_FILE="/input/role1.properties /input/role2.properties" \
+  -e ONTOP_DEV_MODE="false false" \
+  ghcr.io/eclipse-tractusx/knowledge-agents/provisioning-agent:1.9.5-SNAPSHOT
+````
+
+Accessing entities spanning two schemas using the first role/endpoint delivers a greater count
+
+```console
+curl --location --request POST 'http://localhost:8080/sparql' \
+--header 'Content-Type: application/sparql-query' \
+--header 'Accept: application/json' \
+--data-raw 'PREFIX : <http://example.org/voc#>
+
+SELECT (COUNT(DISTINCT ?x) as ?count)
+WHERE {
+   ?x a :Course .
+}'
+```
+
+```json
+{
+  "head" : {
+    "vars" : [
+      "count"
+    ]
+  },
+  "results" : {
+    "bindings" : [
+      {
+        "count" : {
+          "datatype" : "http://www.w3.org/2001/XMLSchema#integer",
+          "type" : "literal",
+          "value" : "12"
+        }
+      }
+    ]
+  }
+}
+```
+
+Accessing entities using the restricted role/endpoint delivers a smaller count
+
+```console
+curl --location --request POST 'http://localhost:8082/sparql' \
+--header 'Content-Type: application/sparql-query' \
+--header 'Accept: application/json' \
+--data-raw 'PREFIX : <http://example.org/voc#>
+
+SELECT (COUNT(DISTINCT ?x) as ?count)
+WHERE {
+   ?x a :Course .
+}'
+```
+
+```json
+{
+  "head" : {
+    "vars" : [
+      "count"
+    ]
+  },
+  "results" : {
+    "bindings" : [
+      {
+        "count" : {
+          "datatype" : "http://www.w3.org/2001/XMLSchema#integer",
+          "type" : "literal",
+          "value" : "6"
+        }
+      }
+    ]
+  }
+}
+```
+
+### Helm
+
+A helm chart for deploying the remoting agent can be found under [this folder](../charts/provisioning-agent).
+
+It can be added to your umbrella chart.yaml by the following snippet
+
+```console
+dependencies:
+  - name: provisioning-agent
+    repository: https://catenax-ng.github.io/product-knowledge/infrastructure
```

Comment:
After the initial contribution you should have GH workflows in place which will release your application and other artifacts like Helm chart.

Do not link to catenax-ng repositories/GH Org.

Reply:
moved to eclipse tractusx helm repo (dev)